assets/node.yaml

```diff
@@ -115,8 +119,15 @@ spec:
           hostPath:
             path: /var/lib/kubelet/plugins_registry/
             type: Directory
-        - name: device-dir
+        - name: efs-state-dir
           hostPath:
-            path: /dev
-            type: Directory
-
+            path: /var/run/efs
+            type: DirectoryOrCreate
+        - name: efs-utils-config
+          hostPath:
+            path: /var/amazon/efs
+            type: DirectoryOrCreate
+        - name: efs-utils-config-legacy
+          hostPath:
+            path: /etc/amazon/efs
+            type: DirectoryOrCreate
```

Comment:
These last 2 have the same `path`, do you know if that's intentional?

If it is, could they be unified?

Reply:
I stared at it for a while upstream too, but there are no two same paths. There is `/var` and `/etc`.